.github/workflows/visual_regression_tests.yml

```diff
@@ -0,0 +1,58 @@
+name: "Visual Tests"
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+  push:
+    paths:
+      - "packages/**"
+    branches:
+      - main
+
+jobs:
+  install-cache:
+    name: Install and cache dependencies
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Commit
+        uses: actions/checkout@v2
+      - name: Cache yarn dependencies and cypress
+        uses: actions/cache@v2
```

Comment:
There's a v3 for the cache action. Do you know if we could use that?

Reply:
Updated to v3.